lib/msf/core/payload/windows/x64/bind_tcp.rb

```diff
@@ -204,6 +205,10 @@ def asm_bind_tcp(opts={})
     ^
 
     asm << asm_send_uuid if include_send_uuid
+    return asm
+  end
+
+  def asm_block_recv(opts={})
 
     asm << %Q^
```

Comment:
Since you encapsulated this in the method, you'll need this to change to `asm = %Q^`
Pretty sure that's why sanity tests are failing on meterpreter/bind_tcp.